docs/_toc.yml

```diff
@@ -48,3 +48,9 @@ parts:
       - file: funding/index.md
       - file: funding/czi-grant-roadmap.md
       - file: funding/czi-grant-completed.md
+  # ------------------------------------------------------
+  # Accessibility audits and improvement strategies
+  - caption: Accessibility audits
+    chapters:
+      - file: audits/index.md
+      - glob: audits/*.md
```

Comment:
This section is now at the bottom of the sidebar, I think all this work deserves a more prominent place, suggest moving it above the "Funded accessibility work" section